enterprise-suite-latest/Makefile

```diff
@@ -23,6 +23,13 @@ include ../$(ES_CHART)/Makefile
 CHART_DIR = ../$(ES_CHART)
 COMPONENTS := $(wildcard ../$(ES_CHART)/*/.)
 SUBCOMPONENTS := $(wildcard ../$(ES_CHART)/*/*/.)
+# Generate dev helm settings from enterprise-suite settings and values-latest.yaml file
+DEV_HELM_SETTINGS := $(DEV_HELM_SETTINGS)$(shell awk '\
+    BEGIN { FS = ": " } \
+    { \
+        settings = settings "," $$1 "=" $$2 \
+    } \
+    END { print settings }' values-latest.yaml)
```

Comment:
Wouldn't it be better to use `yq` vs `awk` hacking?

Reply:
`awk` is my hammer!  What is this new-fangled `yq` that you speak of?

I'll take a look...

Reply:
If it can be solved by yq, we should use, it, we are adding dependency on one more tool

Reply:
I honestly think `awk` is fine for this job.  From what I can tell `yq` is a tool for converting between yaml and JSON.  That's not what I'm doing here.  I'm no `yq` whiz so came up with this:

`yq r -j values-latest.yaml | sed -e 's/[}{]//g' -e 's/"//g' -e 's/:/=/g'`

Is that preferable to the `awk` sol'n?  (Maybe it is.)  Is there a better way to do this with `yq`?

Reply:
With respect to dependencies, I'd say that we can pretty much guarantee that `awk` is already installed.  We have to install `yq` in `scripts/setup-tools-for-ubuntu.sh`.  (And that's just one flavor of `yq`, others of which are incompatible.)

Reply:
We already rely on `yq` for creating the enterprise-suite-latest tarball. So installing it isn't an issue. The main advantage of `yq` is it will work regardless of how the yaml file is formatted - you're making assumptions with awk that the file is formatted a certain way. It's also difficult to understand compared to the `yq` syntax. `yq` is the right tool here, it's designed for reading and manipulating yaml files.

Reply:
Okay `yq` isn't as great as `jq` unfortunately... so it makes it a little harder to use `yq` here, admittedly. I wonder if you can use `--set-file`, which I think takes a `yaml` file directly.